src/components/organisms/Badges/Badges.tsx

```diff
@@ -16,7 +16,7 @@ import { AnyVenue, isVenueWithRooms } from "types/venues";
 import { WithId } from "utils/id";
 import { isTruthy, notEmpty } from "utils/types";
 
-import { useWorldEdit } from "hooks/useWorldEdit";
+import { useWorldById } from "hooks/useWorldById";
```

Comment:
Now it is `hooks/worlds/useWorldById`, needs merge down from staging.

Reply:
b18441c